src/lib/tzcode/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ set(TZDATA_FULL_TARGET ${TZCODE_SOURCE_DIR}/${TZDATA_TARGET})
 set(TZCODE_SRC ${PROJECT_SOURCE_DIR}/src/lib/tzcode)
 set(TZLUA_SRC ${PROJECT_SOURCE_DIR}/src/lua)
 
+if(DEFINED ENV{TZDIR})
```

Comment:
We can evaluate `ENV{TZDIR}` only once and then use variable with value. I'll not insist, it's up to you.